NOTICE.txt

```diff
@@ -0,0 +1,9 @@
+play-test-ops
+Copyright 2022 Optum
```

Comment:
Hey @robertcrockett you normally don't add an Optum copyright in the other PRs for OSPO-ification of the OSS. Why is this repo any different?